### PR TITLE
Community and Collection and Notifications Admin

### PR DESCRIPTION
**Pull request by mbarnett**
Still a WIP but feel free to start reviewing it while I finish things off

### CODE REVIEW

File: app/controllers/communities_controller.rb [hunk 1]
```diff
@@ -23,6 +36,19 @@ def create
     redirect_to @community
   end
 
+  def destroy
+    community = Community.find(params[:id])
+    community.unlock_and_fetch_ldp_object do |uo|
+      if uo.destroy
+        flash[:notice] = 'Community deleted'
```
Comment (cwant) [inline app/controllers/communities_controller.rb]: i18n?

File: app/views/communities/edit.haml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+%h2.mt-3 Create Community
```
Comment (cwant) [inline app/views/communities/edit.haml]: ERB up there ^^^, but HAML here?

Comment (mbarnett) [inline app/views/communities/edit.haml]: The original HAML files from the demo are being rewritten into erb as they get touched. All new work is in erb

**Comment by mbarnett** [comment]
This is (finally) done

**Comment by cwant** [comment]
Travis not happy? (I also had test errors when I checked out the branch recently)

**Comment by mbarnett** [comment]
Weird, it Works on My Machine™...

**Comment by murny** [comment]
Trying merging with master and running the tests again? Might be something in master doing weird things with this branch

File: app/views/collections/show.haml [hunk 3]
```diff
@@ -9,3 +9,5 @@
 					= link_to w.title, work_path(w), class: 'list-group-item list-group-item-action'
 			- else
 				There are no works in this Collection
+
+= link_to 'Edit', edit_community_collection_path(@community, @collection), class:'btn btn-secondary'
```
Comment (mbarnett) [inline app/views/collections/show.haml]: Didn't we change the line length setting?

**Comment by mbarnett** [comment]
WTF. It's complaining about an argument being passed that is definitely no longer being passed in this commit.

**Comment by cwant** [comment]
Looks like I have ualbertalib:collection_admin checked out not mbarnett:collection_admin. Will figure this out and try again.

**Comment by cwant** [comment]
I think Travis is also checking out the wrong repo.

**Comment by mbarnett** [comment]
I just brought that up to date with my fork's branch. I'm going to close this and try reopening off the ualberta branch, see if that does less mysterious things

**Pull request closed without merging**